### DISCUSSION TITLE
Release 0.4.15 "yanked" ?

### DISCUSSION THREAD

**Discussion by GaetanLepage** [jax-ml/jax#17380] [General] [answered]
Hello,

I am working on the packaging of jax on the Nix ecosystem.
The latest jax(lib) release (v0.4.15) is [marked as "yanked" on pypi](https://pypi.org/project/jaxlib/0.4.15/).
Will this then be cancelled or this is just a temporary warning ? Should we wait for a following release ?

Thank you very much for any clarification and most of all for working on this amazing project !

**Comment by jakevdp** [comment, marked as answer]
Yes, we had to yank 0.4.15 shortly after release, because of an issue discovered with the release. See https://github.com/google/jax/issues/17371. The next release will be 0.4.16.

For what it's worth, this happens occasionally, as you can see by perusing the project history: https://pypi.org/project/jax/#history

**Reply by GaetanLepage** [reply]
Thanks for the reply !
Indeed, I encountered the [".dev" version "bug"](https://github.com/google/jax/commit/e01f2617b85c5bdffc5ffb60b3d8d8ca9519a1f3) while building from source.
Setting `JAX[LIB]_RELEASE=1` seems to have fixed the issue.

Anyway, we will then wait for 0.4.16 and update directly to it !
